sound/soc/sof/ipc4.c

```diff
@@ -617,7 +617,13 @@ static int sof_ipc4_set_core_state(struct snd_sof_dev *sdev, int core_idx, bool
 	return sof_ipc4_tx_msg(sdev, &msg, msg.data_size, NULL, 0, false);
 }
 
+static int sof_ipc4_ctx_save(struct snd_sof_dev *sdev)
+{
+	return sof_ipc4_set_core_state(sdev, SOF_DSP_PRIMARY_CORE, false);
+}
+
 static const struct sof_ipc_pm_ops ipc4_pm_ops = {
+	.ctx_save = sof_ipc4_ctx_save,
```

Comment:
it's unclear why we don't have a restore, but that comment can be added later in a fixup patch.

Reply:
OK, I will add a fixup to add a comment.

Sorry, for some reason I have missed your comments and failed to address them in time.